### PR TITLE
added April relnotes

### PR DESCRIPTION
**Pull request by bromeg**
Need updated UID for the Standalone page (from #792 ) and version number of release.

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/fde6a24c23ac225cf9e65e7e123a94a0d1d4aee8/articles/relnotes/index.md) | 76 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd1308ae-39a0-4158-b74d-f5d59acd621d_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 1]
```diff
@@ -16,6 +16,15 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.xx.xxxx.xxxx
+
+- New support for standalone Q# executables, which no longer require a C# or Python host file. For more information on getting started with Q# standalone, see [here]()
```
Comment (bettinaheim) [inline articles/relnotes/index.md:21]:
```suggestion
- New support for Q# command line applications, which no longer require a C# or Python host file. For more information on getting started with Q# command line applications, see [here]()
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/e644acdfafaef9e154729ff605d6b6d873ddc3e8/articles/relnotes/index.md) | 75 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/df4e51ee-6e80-4905-9e1e-da9a6559afc2_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 2]
```diff
@@ -16,6 +16,15 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
 
+## Version 0.xx.xxxx.xxxx
+
+- New support for Q# command line applications, which no longer require a C# or Python host file. For more information on getting started with Q# command line applications, see [here]()
+
+
+*Release date: March 26th, 2020*
+
+This release contains the following:
+
```
Comment (KittyYeungQ) [inline articles/relnotes/index.md:27]:
```suggestion
## Version 0.xx.xxxx.xxxx

- New support for Q# command line applications, which no longer require a C# or Python host file. For more information on getting started with Q# command line applications, see [here](xref:microsoft.quantum.install.standalone)

We enable Q# programs to be run without a classical driver.  See [here](xref:microsoft.quantum.install.standalone) for information on how you can build and run your standalone Q# program either within Visual Studio or VS Code, or from the command line without specifying a Python, C# or other .NET language classical driver.  Standalone Q# offers a great getting-started experience, allowing the new user to focus on the quantum computing code with Q#.  See the updated quantum random number generator Quickstart[xref:microsoft.quantum.quickstarts.qrng]  for an example.  Python and .NET language drivers continue to be supported.  

*Release date: March 26th, 2020*

This release contains the following:

```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/22e71f75be972889cc985006a159f82e6fcaab4d/articles/relnotes/index.md) | 75 | 41 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bd3748bf-4385-4086-be0a-e792909f9bcb_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/5020ef27387055cab961c0835abc72fc1507b423/articles/relnotes/index.md) | 75 | 41 | 22 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a21a58a7-2524-4d5c-9ac2-5ee707880cd0_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 3]
```diff
@@ -16,6 +16,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.11.2004.2825
+
+- New support for Q# command line applications, which no longer require a C# or Python host file. For more information on getting started with Q# command line applications, see [here](xref:microsoft.quantum.install.standalone)
+- Updated quantum random number generator quickstart to no longer require C# or Python host fil. See the updated  [Quickstart](xref:microsoft.quantum.quickstarts.qrng)
```
Comment (rmshaffer) [inline articles/relnotes/index.md:22]:
```suggestion
- Updated quantum random number generator quickstart to no longer require a C# or Python host file. See the updated  [Quickstart](xref:microsoft.quantum.quickstarts.qrng)
```

File: articles/relnotes/index.md [hunk 4]
```diff
@@ -16,6 +16,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
+- Performance improvements to IQ# Docker images
+
+*Release date: March 26th, 2020*
+
+This release contains the following:
```
Comment (rmshaffer) [inline articles/relnotes/index.md:27]: Should this date be updated, and these lines moved above the list of notes?

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/f9df76cac6b2f3b0ceb488214ab279751085dd52/articles/relnotes/index.md) | 75 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/92ea5de1-a080-400c-9e00-46b053c9be5b_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 5]
```diff
@@ -16,6 +16,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
...
 
+## Version 0.11.2004.2825
+
+- New support for Q# command line applications, which no longer require a C# or Python host file. For more information on getting started with Q# command line applications, see [here](xref:microsoft.quantum.install.standalone)
+- Updated quantum random number generator quickstart to no longer require a C# or Python host file. See the updated  [Quickstart](xref:microsoft.quantum.quickstarts.qrng)
+- Performance improvements to IQ# Docker images
+
+*Release date: March 26th, 2020*
+
+This release contains the following:
```
Comment (KittyYeungQ) [inline articles/relnotes/index.md:27]:
```suggestion
## Version 0.11.2004.2825

*Release date: April 29th, 2020*

This release contains the following:

- New support for Q# command line applications, which no longer require a C# or Python host file. For more information on getting started with Q# command line applications, see [here](xref:microsoft.quantum.install.standalone)
- Updated quantum random number generator quickstart to no longer require a C# or Python host file. See the updated  [Quickstart](xref:microsoft.quantum.quickstarts.qrng)
- Performance improvements to IQ# Docker images

```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/163c2f98d8b924de5b1bddbf8af79c78534c0a3c/articles/relnotes/index.md) | 75 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/27146800-a41f-4768-8142-eb5e541cee28_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/163c2f98d8b924de5b1bddbf8af79c78534c0a3c/articles/relnotes/index.md) | 75 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/bf5ebd81-028e-4828-a9f9-8e80802803ad_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 6]
```diff
@@ -16,6 +16,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
+## Version 0.11.2004.2825
+
+*Release date: April 29th, 2020*
```
Comment (efratshabtai) [inline articles/relnotes/index.md:21]: will probably be April 30

Comment (KittyYeungQ) [inline articles/relnotes/index.md:21]:
```suggestion
*Release date: April 30th, 2020*
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/269b3198a40a9f9d1581a851db91b085b3f2b901/articles/relnotes/index.md) | 75 | 41 | 21 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a5ee270f-6f9e-4522-90f4-c95692aa9f6c_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**